apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html

```diff
@@ -8,6 +8,7 @@
   [size]="size"
   [appearance]="appearance"
   [ngClass]="{ fullWidth: isMobileWidthFull }"
+  class="join-button"
```

Comment:
class should be join-community

whenever you're accessing this in code, you could do button.join-community